.github/actions/paths-filter/action.yml

```diff
@@ -11,8 +11,8 @@ outputs:
         github.event_name == 'push' ||
         steps.filter-common.outputs.github-actions-change == 'true'
       }}
-  java-unit-tests:
-    description: Output whether `unit-tests` (java) should be run based on GitHub event and files changed
+  java-code-changes:
+    description: Output whether `java code changes` have been detected
```

Comment:
the reason why I included this "should be run based on GitHub event and files changed" blurb is that this condition also always is true for `push` events and people could assume that is wrong - it is probably a nitpick but just to provide context

Reply:
Not sure what you mean with the comment :) Should I add this back again?

Reply:
I'm not sure myself either what I mean...

I think I'd prefer "Output whether jobs depending on Java code changes should be run based on GitHub event and files changed" here :)

Reply:
How about: 

> Output whether jobs depending on Java code changes should be run, related checks are based on GitHub events and java files changes

Reply:
Fine by me :)